### DISCUSSION TITLE
Can we use tailwind instead of using css modules

### DISCUSSION THREAD

**Discussion by MahirAlam** [mantinedev/mantine#5121] [Feature requests]
I am using tailwind for most of my project then I found mantine and it's kind a good but every time creating a css file is a big pain

So I am thinking is there a way to combine tailwind and mantine natively?

like when we use

`className={classes.header}`

we can use

`className="h-4"`

and some how use the functions

`rem(1px) solid light-dark(var(--mantine-color-gray-3), var(--mantine-color-dark-4))  e.t.c`

in tailwind classes?

**Comment by rtivital** [comment]
You can use any third-party tools with Mantine components. More information here – https://mantine.dev/styles/css-modules/#styling-mantine-components-without-css-modules

**Reply by MahirAlam** [reply]
yeah but it will be very helpful if you can add tailwind beside css modules?

Tailwind is the most popular CSS framework I know
beside I don't like plain css
and if we use tailwind beside mantine we miss a lot of features of them both

like,
functions of mantine, colors of mintine e.t.c.

**Reply by MahirAlam** [reply]
I think you should consider it because a lot of people use tailwind. And I like mantine ui components and hooks a lot.
So it will be very helpful....

**Reply by JhohannesK** [reply]
I think tailwind should genuinely be added to mantine UI as an option when setting it up with a project.

**Reply by MahirAlam** [reply]
Yeah!
You just told what I had in my mind!!!!!
